docs/en/verifying-signatures.wml

```diff
@@ -9,199 +9,380 @@
+Primary key fingerprint: EF6E 286D DA85 EA2A 4BA7  DE68 4E2C 6E87 9329 8290
+     Subkey fingerprint: A430 0A6B C93C 0877 A445  1486 D148 3FA6 C3C0 7136
     </pre>
-    <p>To verify the signature of the package you downloaded, you will need
+   </article>
+  </div>
+<!-- Linux -->
+  <div>
+   <input id="ac-3-3" name="accordion-3" type="radio" />
+   <article class="ac-os">
+    <h3>Verify with OpenPGP signature on Linux</h3>
+    <p>
+    To verify the signature of the package you downloaded, you will need
     to download the ".asc" file as well. Assuming you downloaded the
-    package and its signature to your Downloads folder, run:</p>
+    package and its signature to your Downloads folder, run:
+    </p>
 
-    <strong>For Mac OS X users</strong>:<br />
-    <pre>gpg --verify ~/Downloads/TorBrowser-<version-torbrowserbundleosx64>-osx64_en-US.dmg{.asc*,}</pre>
+    <p>Change 64 to 32 if you have the 32-bit package:</p>
 
-    <strong>For Linux users</strong> (change 64 to 32 if you have the 32-bit package):<br />
-    <pre>gpg --verify tor-browser-linux64-<version-torbrowserbundlelinux64>_en-US.tar.xz.asc</pre>
+    <pre>
+    $ gpg --verify tor-browser-linux64-<version-torbrowserbundlelinux64>_en-US.tar.xz.asc
+    </pre>
 
     <p>The output should say "Good signature":</p>
 
     <pre>
 gpg: assuming signed data in 'tor-browser-linux64-<version-torbrowserbundlelinux64>_en-US.tar.xz'
 gpg: Signature made Wed 15 Nov 2017 05:52:38 PM CET
 gpg:                using RSA key 0xD1483FA6C3C07136
-gpg: Good signature from "Tor Browser Developers (signing key) <torbrowser@torproject.org>" [unknown]
+gpg: Good signature from "Tor Browser Developers (signing key) &lt;torbrowser@torproject.org&gt;" [unknown]
 gpg: WARNING: This key is not certified with a trusted signature!
 gpg:          There is no indication that the signature belongs to the owner.
 Primary key fingerprint: EF6E 286D DA85 EA2A 4BA7  DE68 4E2C 6E87 9329 8290
      Subkey fingerprint: A430 0A6B C93C 0877 A445  1486 D148 3FA6 C3C0 7136
     </pre>
-    <p> Currently valid subkey fingerprints are:
+
+    <p>
+    On <b>Debian</b> for Tor (not Tor Browser) packages, you should read the
+    instructions on
+    <a href="<page docs/debian>#packages">importing these keys to apt</a>.
+    If you're using the <b>RPMs</b> (for Tor, not Tor Browser), you can
+    manually verify the signatures on the RPM packages by
+    <pre>rpm -K filename.rpm</pre>
+    </p>
+
+  </article>
+ </div>
+<!-- END OS specific sections -->
+    <p>
+    Currently valid subkey fingerprints are:
+    </p>
     <pre>
     5242 013F 02AF C851 B1C7  36B8 7017 ADCE F65C 2036
     BA1E E421 BBB4 5263 180E  1FC7 2E1A C68E D408 14E0
-    A430 0A6B C93C 0877 A445  1486 D148 3FA6 C3C0 7136</pre></p>
+    A430 0A6B C93C 0877 A445  1486 D148 3FA6 C3C0 7136
+    </pre>
     <p>
     Notice that there is a warning because you haven't assigned a trust
```

Comment:
Is this section in the right place in the document?

Reply:
it is for all OS. for linux it might be confusing though, moving the debian note up.